components/shared/script/serializable.rs

```diff
@@ -182,3 +182,36 @@ impl BlobImpl {
         &mut self.blob_data
     }
 }
+
+#[derive(Clone, Debug, Deserialize, MallocSizeOf, Serialize)]
+/// A serializable version of the DOMPoint/DOMPointReadOnly interface.
+pub struct DomPoint {
```

Comment:
I think that `DOMPoint`versus `DomPoint` could be pretty confusing to someone reading the code, so I suggest `SerialilzableDomPoint` here as the name.